src/org/traccar/protocol/EelinkProtocolDecoder.java

```diff
@@ -97,28 +114,72 @@ private String decodeAlarm(Short value) {
     }
 
     private void decodeStatus(Position position, int status) {
+
+        // GPS - 0 bit
```

Comment:
They are necessary for clarification to all people, no point in people having to refer back to specs to know what the provider refer to the parts, you use KEY_IGNITION they refer to ACC.